.changeset/ninety-seas-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/cli': patch
+---
+
+Allow users to config their own PI chain
```

Comment:
More descriptive message here please. Something like `Allow users to only configure validators for their chain`